packages/dx-react-scheduler-material-ui/src/templates/editing-menu/container.jsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  container: {
+    position: 'absolute',
+    width: '100%',
+    height: '100%',
+  },
+};
+
+export const ContainerBase = ({
+  children, target, classes, className, ...restProps
+}) => (
+  <div
+    ref={target}
```

Comment:
Seems that `target` name is misleading here because Container is itself a target. Better just use ref forwarding